### DISCUSSION TITLE
Releasing a new version: 0.1.7

### DISCUSSION THREAD

**Discussion by kaplanelad** [loco-rs/loco#169] [Announcements]
🎉 Loco version 0.1.7 was released! Explore the [changelog](https://github.com/loco-rs/loco/blob/master/CHANGELOG.md#v017) to discover exciting new features.

Highlights of this release include:
- Added pretty backtraces
- Serving static assets 
- Support shuttle deployment
- Added ability to specify settings: in config files, which are available in context

Enjoy :)

**Comment by Anakin100100** [comment]
I'm really excited to see you shipping new versions regularly. Keep going :)
